session_0/item.rb

```diff
@@ -23,14 +21,24 @@ def initialize(name: '', price: 0, quantity: 1, category_id: 4,
   # Returns a boolean value whether than item is discounted i.e. the
   # discount deadline has been crossed or not.
   def discounted?
-    raise NotImplementedError # TODO
+    if Time.now > discount_deadline
```

Comment:
So would i get discount, if i make a purchase on discount_deadline ???

Reply:
You will.

Reply:
I'm sorry but I do not understand if you want me to change something here. I think the in this case the condition of equality is a bit ambiguous but I think you should get a discount in that case